platforms/cuda/include/CudaContext.h

```diff
@@ -356,6 +356,12 @@ class OPENMM_EXPORT_COMMON CudaContext : public ComputeContext {
     bool getIsCPU() const {
         return false;
     }
+    /**
+     * Get the tile size of the device being used.
```

Comment:
That isn't what `TILE_SIZE` means.  It's the size of the tiles used in computing nonbonded interactions.  It doesn't have anything to do with a particular device.